services/order.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, User
+from services.movie_session import get_movie_session_by_id
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: datetime = None) -> None:
+    with transaction.atomic():
+        order = Order.objects.create(
+            user=User.objects.get(username=username)
+        )
+
+        if date:
+            order.created_at = date
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session=get_movie_session_by_id(ticket["movie_session"]),
+                order_id=order.id
+            )

```

Comment:
You can try yo use bulk_create to create all tickets at once and not overload db, but its not necessary here, just recommendation)